src/index.d.ts

```diff
@@ -115,7 +116,7 @@ declare namespace preact {
 
 		state: Readonly<S>;
 		props: RenderableProps<P>;
-		context: any;
+		context: C;
```

Comment:
This might be odd when using `contextType`? Not sure. Using `contextType` assigns `this.context` to the current value of the given context provider.

Reply:
You mean that it's possible to assign any `Context` to the static `contextType`? Or that `contextType` could be dynamically switched?